lib/helpers.js

```diff
@@ -47,18 +47,34 @@ const getCartId = async (request, authorization, body) => {
   return responseBody._id
 }
 
-const createUser = async page => {
-  await page.type('data-testid=nome', faker.name.findName())
-  await page.type('data-testid=email', faker.internet.email())
-  await page.type('data-testid=password', faker.internet.password())
+const login = async (page, email, password) => {
+  await page.type('data-testid=email', email)
+  await page.type('data-testid=senha', password)
+
+  await page.click('data-testid=entrar')
+}
+
+const createUser = async (page, name, email, password) => {
+  await page.type('data-testid=nome', name)
+  await page.type('data-testid=email', email)
+  await page.type('data-testid=password', password)
   await page.click('data-testid=checkbox')
 
   await page.click('data-testid=cadastrar')
 }
 
-const openHomepage = async page => {
+const createProduct = async (page, name, price, description, qt) => {
+  await page.type('data-testid=nome', name)
+  await page.type('data-testid=preco', price)
+  await page.type('data-testid=descricao', description)
+  await page.type('data-testid=quantity', qt)
+
+  await page.click('data-testid=cadastarProdutos')
+}
+
+const openHomepage = async (page, name, email, password) => {
   await page.goto('/cadastrarusuarios')
-  await createUser(page)
+  await createUser(page, name, email, password)
```

Comment:
@uLucasFraga too many arguments, we can simply take a `user` and then get the required data from it

```suggestion
  await createUser(page, user)
```